src/components/structures/login/Login.js

```diff
@@ -20,12 +21,11 @@ limitations under the License.
 import React from 'react';
 import PropTypes from 'prop-types';
 import { _t } from '../../../languageHandler';
-import * as languageHandler from '../../../languageHandler';
 import sdk from '../../../index';
 import Login from '../../../Login';
-import PlatformPeg from '../../../PlatformPeg';
 import SdkConfig from '../../../SdkConfig';
-import SettingsStore, {SettingLevel} from "../../../settings/SettingsStore";
+import SettingsStore from "../../../settings/SettingsStore";
+import makeLanguageSelector from "./LanguageSelector";
```

Comment:
Instead of importing this directly, we should be using `sdk.getComponent`

Reply:
Fair enough

Re this there's actually more instances of direct imports of `<AccessibleButton />` than skinned imports.
29 vs 16 :(

Reply:
>there's actually more instances of direct imports of <AccessibleButton /> than skinned imports.

Yeah, for some reason it's almost systematically been done incorrectly for AccessibleButton. 

It's unfortunate that we're paying the price for the component drop-in system without any benefits.